manager/integration/tests/test_uninstall.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import os
+import time
+
+from common import SETTING_DELETING_CONFIRMATION_FLAG
+from common import batch_v1_api, core_api, get_longhorn_api_client # NOQA
+from common import RETRY_COUNTS, RETRY_INTERVAL_LONG
+
+@pytest.mark.uninstall   # NOQA
+def test_uninstall(batch_v1_api, core_api): # NOQA
+
+    client = get_longhorn_api_client()
+    lh_raw = "https://raw.githubusercontent.com/longhorn/longhorn"
+
+    longhorn_version = os.getenv("LONGHORN_REPO_BRANCH")
+
+    if longhorn_version == "master" or longhorn_version == "v1.4.x":
+        delete_confirmtion_flag = \
+            client.by_id_setting(SETTING_DELETING_CONFIRMATION_FLAG)
+        setting = client.update(delete_confirmtion_flag, value="true")
+        assert setting.name == SETTING_DELETING_CONFIRMATION_FLAG
+        assert setting.value == "true"
```

Comment:
You can just try to see if the setting exists to determine if the testing version has to the deletion flag support instead of hard-coding the version check.